<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PolarGraphWindow</class>
 <widget class="QMainWindow" name="PolarGraphWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Polar plotter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>128</width>
     <height>128</height>
    </size>
   </property>
   <widget class="QLabel" name="plotLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>819</width>
      <height>559</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;Вид</string>
    </property>
    <addaction name="dataPanelAction"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>&amp;Справка</string>
    </property>
    <addaction name="aboutAction"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;Файл</string>
    </property>
    <addaction name="exitAction"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="exitAction">
   <property name="text">
    <string>&amp;Выход</string>
   </property>
  </action>
  <action name="dataPanelAction">
   <property name="text">
    <string>Панель &amp;данных...</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>О &amp;программе</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
